doc/replication.md

```diff
@@ -68,65 +66,62 @@ devices. This web service needs to:
 ### Replication on the Device
 
 From the device side, replication is straightforward. You can replicate from a
-local datastore to a remote database, from a remote database to a local
-datastore, or both ways to implement synchronisation.
+local document store to a remote database, from a remote database to a local
+document store, or both ways to implement synchronisation.
 
 Replications are set up in code on a device. Use `ReplicatorBuilder` to create
 a pre-configured `Replicator` object. Each `Replicator`
 object can be assigned a `Listener` to receive message when replication
 completes or encounters an error.
 
+The replication examples below are 'one-shot' and will run once until
+completion, at which point the local and remote databases will be
+synchronised. [Replication policies](replication-policies.md) allow
+for more complex "policies" to be executed, eg "only replicate when on
```

Comment:
as per discussion on the other doc change yesterday, we should change `eg` to `for example`